README.md

```diff
@@ -1 +1,39 @@
-# salesforcedx-vscode-automation-tests
+# Salesforce Extensions for VS Code
+[![Commitizen friendly](https://img.shields.io/badge/commitizen-friendly-brightgreen.svg)](http://commitizen.github.io/cz-cli/)
+
+## Introduction
+This repository contains the source code for the automation tests for the Salesforce Extensions for VS Code.
+
+### WDIO VSCode Service
+This project is based on `WDIO VSCode Service, available at https://github.com/webdriverio-community/wdio-vscode-service
+
+### Getting Started
+If you are interested in contributing, please take a look at the [CONTRIBUTING](CONTRIBUTING.md) guide.
+
+After cloning this repo, you will also need to clone https://github.com/forcedotcom/salesforcedx-vscode. The default topography is for `salesforcedx-vscode` and `salesforcedx-vscode-automation-tests` to reside side-by-side in the same parent location.
+
+To install the dependencies, run `npm install`. You do not need to compile - when running the e2e automation tests the code is dynamically compiled.
+
+For the VSCode extension (`{location}/salesforcedx-vscode`), you will need to run `npm install` but you do not need to compile or perform any other steps.
```

Comment:
not necessary maybe, since we don't have it in the main VSCE repo, but would it be helpful to have one more task to be able to run this from command palette?

Reply:
Done - added the "Install Node Dependencies For Project" task